### ISSUE TITLE
Workflow doesn't document ONNV_BUILDNUM

### ISSUE THREAD

**Issue by trisk** [illumos/dev-guide#22]
Current doc doesn't mention that ONNV_BUILDNUM may need incrementing in most or all install target environments.

**Comment by rmustacc** [comment]
Can you summarize the current rules one should use in determining how to set it, or ultimately, am I better off just doing a per-distro section here?

**Comment by trisk** [comment]
It's technically only relevant to distros that use the pkg(5) metadata for upgrades. My previous suggestion on the wiki was:
"Run:
```
pkg info osnet-incorporation | grep Branch:
```
If you see, for example, "Branch: 0.151.1", you must choose a number greater than the part after the leading "0.". For example, add
`export ONNV_BUILDNUM=152`
to the illumos.sh script."

I never determined if ONNV_BUILDNUM could have decimals as well.

A large sequence of digits such as YYYYMMDD is also likely to be fine for all existing distros and probably more useful, though this may bring up the gate versioning arguments.
